packages/muelu/test/unit_tests/Hierarchy.cpp

```diff
@@ -1567,11 +1567,9 @@ namespace MueLuTests {
     RCP<RealValuedMultiVector> coordinates = Galeri::Xpetra::Utils::CreateCartesianCoordinates<SC,LO,GO,Map,RealValuedMultiVector>("1D", A->getRowMap(), galeriList);
 
     Teuchos::ParameterList paramList;
-#if defined(HAVE_MPI) && defined(HAVE_MUELU_ZOLTAN2)
```

Comment:
Just double-checking: this used to be `#ifdef HAVE_MPI`. Doesn't this throw now when MPI=OFF?

Reply:
I double-checked, and currently we don't actually throw if MPI=OFF. We simply turn repartition off and print a warning.
```
#ifdef HAVE_MPI
...
#else
      paramList.set("repartition: enable",false);
      this->GetOStream(Warnings0) << "No repartitioning available for a serial run\n";
#endif
```